controllers/C_Document.class.php

```diff
@@ -1014,7 +1014,7 @@ function update_action_process(string $patient_id = null, $document_id)
             }
 
             if (preg_match('/^\d\d\d\d-\d+-\d+$/', $docdate)) {
-                $docdate = "'$docdate'";
+                $docdate = "$docdate";
             } else {
                 $docdate = "NULL";
```

Comment:
Maybe this is a good time as any to get rid of NULL dates. Specific restrictions from the documentation are:

> The value inserted must not have a zero for the month part, the day part, or both parts. This is enforced beginning with MySQL 8.0.22, regardless of the server SQL mode setting.

Reply:
@mdsupport , how would `NULL` dates violate the new enforcement of no zeroes?

Reply:
How about the creation date of the document? It would be in the metadata, no?